tests/code-splitting/subtests/dynamic-import-preloading/parcel/src/utils.js

```diff
@@ -0,0 +1,316 @@
+import { exclaim, reallyBigString, reallyBigString1 } from './exclaim.js';
+
+export function logCaps(msg) {
+  if (!msg) return;
+  console.log(msg.toUpperCase());
+}
+
+export function shout(msg) {
+  logCaps(exclaim(msg));
+  console.log(reallyBigString);
+  console.log(reallyBigString1);
+  console.log(reallyBigString3);
+}
+
+export const reallyBigString3 = `
```

Comment:
I think this is not needed

Reply:
My bad...I accidentally committed my experimental code. I have removed it now.